articles/quickstart/native/ios-swift/interactive.md

```diff
@@ -214,9 +214,9 @@ Verify that you can access the `email`, `picture`, or any other [claim](/secure/
 :::
 
 :::checkpoint-failure
-Sorry about that. Here's a couple things to double check:
-- Did you import the `JWTDecode` module in the file where you're calling the `decode(jwt:)` method?
-- Make sure the claims are spelled correctly.
+If your application did not launch successfully:
```

Comment:
Same as above: the app launch is not what could fail here. The app will launch successfully.